web-console/src/views/load-data-view/load-data-view.tsx

```diff
@@ -1076,11 +1076,21 @@ export class LoadDataView extends React.PureComponent<LoadDataViewProps, LoadDat
             <LearnMore href={getIngestionDocLink(spec)} />
           </Callout>
           {ingestionComboType ? (
-            <AutoForm
-              fields={getIoConfigFormFields(ingestionComboType)}
-              model={ioConfig}
-              onChange={c => this.updateSpecPreview(deepSet(spec, 'spec.ioConfig', c))}
-            />
+            <>
+              <AutoForm
+                fields={getIoConfigFormFields(ingestionComboType)}
+                model={ioConfig}
+                onChange={c => this.updateSpecPreview(deepSet(spec, 'spec.ioConfig', c))}
+              />
+              {deepGet(spec, 'spec.ioConfig.inputSource.properties.secretAccessKey.password') && (
+                <FormGroup>
+                  <Callout intent={Intent.WARNING}>
+                    Inlining the access key into the ingestion spec is dangerous as it might appear
+                    in server log files and can be seen by anyone accessing this console.
```

Comment:
Trying out github's multi-line suggestion 🤞with Steve's suggested re-wording 
```suggestion
                    This key will be visible to anyone accessing this console and may appear in server logs. 
                    For production scenarios, use of a more secure secret key type is strongly recommended.
```